### PR TITLE
Better explanation of user creation

### PR DESCRIPTION
**Pull request by mariatechmaniac**
Explain that no params are required
Help them understand that those users are mapped to their users
Give examples of other fields they can use in the obj

### CODE REVIEW

File: _documentation/conversation/building-blocks/user/create-user.md [hunk 1]
```diff
@@ -8,13 +8,19 @@ In this building block you will see how to create a User.
 
 ## Example
 
-Ensure the following variables are set to your required values using any convenient method:
+In order to create a user, no parameters are required in the request's payload.
```
Comment (mheap) [inline _documentation/conversation/building-blocks/user/create-user.md]: What's our recommended way to create users? Building blocks should promote best practices.

Some libraries will have username and display name as required parameters, so we should decide what experience we want to promote

Comment (mariatechmaniac) [reply _documentation/conversation/building-blocks/user/create-user.md]: @mheap 
i have rewritten it

**Comment by tbedford** [comment]
Maria thanks for the input. It feels like this additional material would be better placed in the API Ref as that is the definitive documentation for the call. Building Blocks are never meant to fully document a call and all parameters, they are only to show a single use case. I will also add some inline comments.

File: _documentation/conversation/building-blocks/user/create-user.md [hunk 2]
```diff
@@ -8,13 +8,21 @@ In this building block you will see how to create a User.
...
+You can create a user by defining the following variables as part of the request:
 
 Key | Description
 -- | --
 `USER_NAME` | The unique name of the User.
 `USER_DISPLAY_NAME` | The display name of the User.
 
+Typically, you can utilise the `USER_NAME` as a means to map your Nexmo User to your own system's users.
+If that is not passed in the request, we will generate and populate the field with a unique ID.
+The `USER_DISPLAY_NAME` can be used for your UI.
```
Comment (tbedford) [inline _documentation/conversation/building-blocks/user/create-user.md]: We should be describing the individual parameters in the API Reference as that is the definitive guide to all calls/parameters. This is true for all calls, not just create user.

File: _documentation/conversation/building-blocks/user/create-user.md [hunk 3]
```diff
@@ -8,13 +8,21 @@ In this building block you will see how to create a User.
...
+
+You can enrich the user object by populating even more parameters like channel details(ex. phone number) and image_url to point to a thumbnail. You can find those in the API Ref[https://developer.nexmo.com/api/conversation#createUser] though in general, no parameters are required in the request's payload in order to create your users.
```
Comment (tbedford) [inline _documentation/conversation/building-blocks/user/create-user.md]: More text required for the parameters and their application in the API Reference. The API Reference is also clear on which parameters are "Required" and which are optional.

**Comment by mheap** [comment]
Closing in favour of a CS API review

**Pull request closed without merging**